healtcheck.go

```diff
@@ -0,0 +1,52 @@
+package server
+
+import (
+	"github.com/juju/errgo"
+)
+
+const (
+	StatusHealthy   = "healthy"
+	StatusUnhealthy = "unhealthy"
+)
+
+type HealthInfo struct {
```

Comment:
There is no limit. That way it can support any service structure. A service can have infinite? backends.